elwood/elwood.py

```diff
@@ -211,30 +211,15 @@ def rescale_dataframe_time(
     )
 
 
-def regrid_dataframe_geo(dataframe, geo_columns, time_column, scale_multi, scale=None):
-    """Regrids a dataframe with detectable geo-resolution
-
-    Args:
-        dataframe (_type_): _description_
-        geo_columns (_type_): _description_
-        scale_multi (_type_): _description_
-    """
-
-    return regrid_dataframe(
-        dataframe=dataframe,
-        geo_columns=geo_columns,
-        time_column=time_column,
-        scale_multi=scale_multi,
-        scale=scale,
-    )
+regrid_dataframe_geo = regrid_dataframe
```

Comment:
If we are doing this on one of the entry points we will probably want to do this on all of the entry-points, but I think this is less human-readable as a library entry-point than what we had previously.